bl-kernel/abstract/dblist.class.php

```diff
@@ -44,6 +44,10 @@ public function getList($key, $pageNumber, $amountOfItems)
 			return $list;
 		}
 
+        if (count($list) < $amountOfItems) {			
```

Comment:
if we have only a few items, and the user requests `pageNumber = 2` is going to returns the same as `pageNumber = 1`.